dss-api.yml

```diff
@@ -2257,6 +2257,194 @@ paths:
                     description: Detailed error message.
                 required:
                   - code
+  /events:
+    get:
+      operationId: dss.api.events.list_events
+      summary: Replay events
+      description: >
+        Return urls where event data is available, with manifest of contents.
+      parameters:
+        - name: from_date
+          in: query
+          description: Timestamp to begin replaying events, in DSS_VERSION format.  If this is not provided, replay from the earliest event.
+          required: false
+          type: string
+        - name: to_date
+          in: query
+          description: Timestamp to stop replaying events, in DSS_VERSION format.  If this is not provided, replay to the latest event.
+          required: false
+          type: string
+        - name: replica
+          in: query
+          description: Replica to fetch from.
+          required: true
+          type: string
+          enum: [aws, gcp]
+        - name: per_page
+          in: query
+          description: Max number of results to return per page.
+          required: false
+          type: integer
+          format: int32
+          minimum: 1
+          maximum: 10
+          default: 1
```

Comment:
Should this be increased?

Reply:
The journals will typically be large files that are streamed from S3, so 1 seems reasonable for now. This may change if/when we add a CLI operation to facilitate event streaming.